apps/client/src/app/pages/portfolio/x-ray/x-ray-page.component.scss

```diff
@@ -1,3 +1,18 @@
 :host {
   display: block;
+
+  .icon-container {
+    background-color: rgba(var(--dark-primary-text), 0.05);
```

Comment:
This does not seem to work. Please change to
```
background-color: rgba(var(--palette-foreground-base), 0.02);
```

(also in `rule.component.scss`)